netbout-client/src/main/java/com/netbout/client/RtAttachment.java

```diff
@@ -119,7 +119,17 @@ public boolean unseen() throws IOException {
 
     @Override
     public Date date() throws IOException {
-        return new Date();
+        return new Date(
```

Comment:
@original-brownbear I am not sure of what you mean here, do you mean that there is duplication in my code regrading `RtAttachment.ctype()`? Or the same duplications as in `RtAttachment.etag()`, `RtAttachment.unseen()`, `RtAttachment.author()`? If it is not specific to my code, you can create a bug about it, I wouldn't consider it specific to this task, nor a puzzle, since there is nothing incomplete in the code in here.